tests/unit/Alteration.spec.ts

```diff
@@ -410,7 +409,7 @@ describe('Alteration component', () => {
 
   it('updates the fees with priority and waive fees for staff payment changes', async () => {
     store.state.stateModel.staffPayment = {
-      option: StaffPaymentOptions.NO_FEE,
+      option: 0, // no fee
```

Comment:
Tests should not use enums. This makes them more fragile -- for example, an error in the enum wouldn't be caught if the test verified the enum itself.

Reply:
Ah, that's a good point.  I'm pretty sure I've been using enums in my tests.  I'll follow your example.